src/common/post.ts

```diff
@@ -145,6 +145,7 @@ export const createSquadWelcomePost = async (
       banned: true,
       private: true,
       visible: true,
+      showOnFeed: false,
```

Comment:
Currently `flags` is not used as a primary source so we should add `showOnFeed` as a main property of the object